mbed_tools_ci/assert_news.py

```diff
@@ -158,7 +149,7 @@ def verify(self) -> None:
         logger.info(':: Checking news files format')
         for news_file in added_news:
             NewsFileValidator(os.path.realpath(
-                os.path.join(project_root, news_file))).verify()
+                os.path.join(project_root, news_file))).validate()
```

Comment:
I suggest we use `pathlib` for all path manipulation instead of strings.

Reply:
We definitely should. I didn't change this particular line as it was outside of the scope of testing. Do you mind if it happens at a later stage?

Reply:
Yeah, if it's outside the scope of this PR I don't mind leaving it until later.